airflow/operators/python.py

```diff
@@ -440,6 +440,7 @@ def __init__(
         expect_airflow: bool = True,
         skip_on_exit_code: int | Container[int] | None = None,
         use_dill: bool = False,
+        use_airflow_context: bool = False,
```

Comment:
I'd be happy for a related docstring in `PythonVirtualenvOperator` and some more documentation in `docs/apache-airflow/howto/operator/python.rst`

Reply:
This is the hardest area for me.
I have a hard time communicating without a translator.
I could work on documentation, but it wouldn't be a good product.

Reply:
> This is the hardest area for me.
> I have a hard time communicating without a translator.
> I could work on documentation, but it wouldn't be a good product.

No worries, do you want to give it a shot, and I'll rephrase afterward? Or, I could write it for you entirely if you want.

Let me know what works best for you, and I'll help :)

Reply:
I tried as you suggested.
I added 2 examples (decorator, classic) and added the relevant content to `python.rst`.

Let me know if there are any semantic issues, grammatical issues, or if it doesn't work in any way. I'll check it out and fix it.

Reply:
Looks amazing!